### ISSUE TITLE
One computer never uploads to peers, one does

### ISSUE THREAD

**Issue by deckname** [Chocobozzz/PeerTube#3481]
**Describe the current behavior**
I have two computers in the same network. Both use Firefox. On one I never see peertube upload to other peers, on the other I frequently can see it share data. 

**Describe the expected behavior**
I would expect both computers to upload to peers sometimes. 

**Additional information**

* PeerTube instance:
  * URL: https://peertube.social, https://share.tube
  * version: 2.4.0, 2.4.0

* Browser name, version and platforms on which you could reproduce the bug: Firefox, 84.0, both have uBlock Origin installed

**Comment by Chocobozzz** [comment]
Do you have logs in the console from the web browser that never share data?

**Comment by deckname** [comment]
Sorry I only looked at the Network tab, not console. Here it clearly says this: 

```
WebRTC: ICE failed, add a TURN server and see about:webrtc for more details 2
Error: Connection error: Ice connection failed.
    r https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    f https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    emit https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    _destroy https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    destroy https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    _onIceStateChange https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    oniceconnectionstatechange https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
6-es2015.ad35dca256d781e56b81.js:1:182770
Error: Connection error: Ice connection failed.
    r https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    f https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    emit https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    _destroy https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    destroy https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    _onIceStateChange https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    oniceconnectionstatechange https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
6-es2015.ad35dca256d781e56b81.js:1:182770
WebRTC: ICE failed, add a TURN server and see about:webrtc for more details 2
Error: Connection error: Ice connection failed.
    r https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    f https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    emit https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    _destroy https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    destroy https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    _onIceStateChange https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    oniceconnectionstatechange https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
6-es2015.ad35dca256d781e56b81.js:1:182770
WebRTC: ICE failed, add a TURN server and see about:webrtc for more details
Error: Connection error: Ice connection failed.
    r https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    f https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    emit https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    _destroy https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    destroy https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    _onIceStateChange https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
    oniceconnectionstatechange https://share.tube/client/en-US/4-es2015.964e02af2099ff21a03a.js:1
6-es2015.ad35dca256d781e56b81.js:1:182770
Cannot play media. No decoders for requested formats: video/mp4; codecs="avc1.64001f", video/mp4; codecs="avc1.64001f", audio/mp4; codecs="mp4a.40.2", audio/mp4; codecs="mp4a.40.2" 07b020ee-9676-4280-aa75-39a039768764
MouseEvent.mozPressure is deprecated. Use PointerEvent.pressure instead.
```

**Comment by deckname** [comment]
Oh, this problem just exists with firefox, not with chromium. Sorry, I just discovered this now. Chromium doesn't report errors in the console and opening the same video in two windows and it starts uploading to the other one.

**Comment by deckname** [comment]
I found the problem! 
On the Firefox that didn't work I had the option `media.peerconnection.ice.proxy_only` manually set to true. I don't remember doing this, maybe it was some addon or whatever, but that was the problem. 
The `WebRTC: ICE failed, add a TURN server and see about:webrtc for more details` errors are very infrequent now, maybe just occur when I reload the page or something. 

I found this, because I stared at `about:webrtc` for a long time, compared it to my working firefox install on a different computer. In the end I discovered that in the very bottom it said `media.peerconnection.ice.proxy_only is set to true`
